tests/integration/run_tests.py

```diff
@@ -340,6 +448,15 @@ async def main():
     LOG.debug("=== Testing getting all objects & pagination ===")
     await test_getting_all_objects_from_schema_works()
 
+    # Test creating, reading, updating and deleting folders
+    LOG.debug("=== Testing basic CRUD folder operations ===")
+    await test_crud_folders_works()
+
+    # Test reading, updating and deleting folders
+    LOG.debug("=== Testing basic CRUD user operations ===")
+    await test_crud_users_works()
+
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(main())
```

Comment:
This because the original causes: `AttributeError: module 'asyncio' has no attribute 'run'` 
when I run this.

Reply:
Yea I realized this was the case but failed to properly understand how `pyenv` operates at the time. Now I do and I reverted back.